node/src/RedisClient.ts

```diff
@@ -215,6 +216,20 @@ export class RedisClient extends BaseClient {
         return this.createWritePromise(createConfigSet(parameters));
     }
 
+    /** Echoes the provided `message` back.
+     * See https://redis.io/commands/echo for more details.
+     *
+     * @param message - The message to be echoed back.
+     * @returns The provided `message`.
+     *
+     * @example
+     *      await client.echo("Glide-for-Redis");
+     *      "Glide-for-Redis"
```

Comment:
please check whats they nodejs way of writing an example - should the response be simply with a newline as you did, or should it be formatted in a different way